constants/constants.go

```diff
@@ -0,0 +1,10 @@
+package constants
+
+const (
+	// ComponentNameKey is the tag key identifying the service
+	ComponentNameKey = "lightstep.component_name"
+	// HostnameKey is the tag key identifying hostname
+	HostnameKey = "lightstep.hostname"
```

Comment:
can we re-use https://github.com/lightstep/lightstep-tracer-go/blob/master/options.go#L41?